src/test/java/org/apache/bcel/verifier/VerifierTestCase.java

```diff
@@ -126,6 +126,16 @@ public void testDefinitionImpl() throws ClassNotFoundException {
         testNestHostWithJavaVersion("com.ibm.wsdl.DefinitionImpl");
     }
 
+    @Test
+    public void testJvmOpCodes() throws ClassNotFoundException {
+    	testDefaultMethodValidation("org.apache.bcel.verifier.tests.JvmOpCodes");
+    }
+
+    @Test
+    public void testObjectBrowser() throws ClassNotFoundException {
+        testDefaultMethodValidation("groovy.inspect.swingui.ObjectBrowser"); // contains SWAP instruction
```

Comment:
This is not maintainable IMO because it's too easy to update Groovy and loose whatever this test thinks it's testing. If you want to test a specific instruction, add a class file and test that.

Reply:
this seems to belong to discontinued maven coordinates though https://search.maven.org/artifact/org.codehaus.groovy/groovy-all-minimal

Reply:
This class groovy.inspect.swingui.ObjectBrowser requires too many other classes. I find keeping jar less worse. Dependabots will not update as it's the end of an old group/artifact.